Tests/TSCBasicTests/ProcessTests.swift

```diff
@@ -42,20 +42,57 @@ class ProcessTests: XCTestCase {
     }
 
     func testPopen() throws {
+        #if os(Windows)
+        let echo = "echo.exe"
```

Comment:
This worries me - there is no `echo.exe` technically.  It is a MSys/2 binary that is part of git and if you add Git to your path, it might show up or it might now.  On the other hand, the vast majority of the t-s-c tests will just abort anyways, so whats a broken test?

The equivalent of echo is echo, which is a shell builtin.